app/views/outbound_webhooks/_fields.html.erb

```diff
@@ -1,5 +1,6 @@
 <%= render 'shared/errors', object: form.object %>
 
+<%= form.input :name, help: "For global webhooks" %>
```

Comment:
only global ones can have names? or can all?

Reply:
local can have names, but does not have to ... I guess the db uniqueness will get weird if people start using them for local ones too ... I'll just drop the index 🤷‍♂